stdlib/testing/testdata/date_month.flux

```diff
@@ -0,0 +1,71 @@
+package testdata_test
+
+import "testing"
+import "date"
+
+option now = () => (2030-01-01T00:00:00Z)
+
+inData = "
+#datatype,string,long,dateTime:RFC3339,string,string,double
+#group,false,false,false,true,true,false
+#default,_result,,,,,
+,result,table,_time,_measurement,_field,_value
+,,0,2018-05-22T19:53:00Z,_m,FF,1
```

Comment:
I agree with @nathanielc on this. Can you vary these so the output has different things?